package.json

```diff
@@ -21,9 +21,11 @@
     "breakjs": "^1.0.0",
     "classnames": "^2.2.5",
     "css-element-queries": "^1.0.1",
+    "enzyme-to-json": "^3.3.3",
```

Comment:
Is that apart of the context?

Reply:
Should be moved back down to devDeps if the upgrade is needed.